### ISSUE TITLE
Add configuration setting to force x265 when getting transcoded streams

### ISSUE THREAD

**Issue by noahhaon** [jellyfin/jellyfin-android#1015]
**Describe the feature you'd like**
For many bandwidth limited cases (mobile data, in particular) you get much better quality for lower bitrates with the x265 codec. As I understand this is a client issue, and jellyfin server will happily transcode to whatever codec the client says it supports. Most modern Intel CPUs also support hardware encoding of x265, so this is something that should have wide support in the server.

The support to force x265 transcoding was added to jellyfin-media-player as of v1.9.0 and it would be wonderful if it could be exposed as an option in the android app as well:

https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.9.0

Thanks for all your work on a great project! :)

**Comment by njmdietrich** [comment]
I agree, it's not even necessary to force HEVC but you can just prefer it. It basically just has to change the order of the list of supported codecs that Jellyfin android sends to the server.

**Comment by gjp1120** [comment]
+1

**Comment by BillCipher-exe** [comment]
currently its inposible to transcode to HEVC beauce only H264 support seem to be hardcoded. see #1612 

edit: @noahhaon found a workaround for me, just hardcoded this bit in the nativeshell.js in the getDeviceProfile function.

        profile.TranscodingProfiles.push({
         "Container": "mp4",
         "Type": "Video",
         "AudioCodec": "aac,mp2,opus,flac",
         "VideoCodec": "av1,hevc,h264,vp9",
         "Context": "Streaming",
         "Protocol": "hls",
         "MaxAudioChannels": "2",
         "MinSegments": "1",
         "BreakOnNonKeyFrames": true
        });
